modules/core/src/main/java/org/apache/ignite/internal/processors/service/IgniteServiceProcessor.java

```diff
@@ -168,6 +168,7 @@ public class IgniteServiceProcessor extends GridProcessorAdapter implements Igni
      * @see ServiceDeploymentManager
      */
     private final ConcurrentMap<IgniteUuid, ServiceInfo> registeredServices = new ConcurrentHashMap<>();
+    private final ConcurrentMap<String, ServiceInfo> registeredServicesByName = new ConcurrentHashMap<>();
```

Comment:
Missing javadoc (build will fail).
Let's mention that this map simply improves lookup by name and should be always in sync with the one above.

Reply:
ok, will do

Reply:
Javadoc has been added.